packages/react-integration/demo-app-ts/src/components/demos/CardDemo/CardDemo.tsx

```diff
@@ -73,25 +75,25 @@ export class CardDemo extends React.Component {
 
   render() {
     const dropdownItems = [
-      <DropdownItem key="link">Link</DropdownItem>,
-      <DropdownItem key="action" component="button">
+      <DropdownItemDeprecated key="link">Link</DropdownItemDeprecated>,
```

Comment:
The dropdown in the toolbar of the Card view demo is not working.  That was preexisting to this PR though. probably broke with overflow PR work...  We can open a follow up issue.

Reply:
I can see whether it's something I can fix as part of the followup to updating usage of deprecated Dropdown to new Dropdown. If it's anything more involved I can open a separate follow up issue.

Reply:
Sounds good. thanks.